src/Turbo/Bridge/Mercure/Resources/assets/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "@symfony/mercure-ux-turbo",
```

Comment:
```suggestion
    "name": "@symfony/ux-turbo-mercure",
```

?

Reply:
using notifier/mailer's style, it should be `mercure-ux-turbo`